README.md

```diff
@@ -27,6 +27,7 @@ Configuru is a library for configuration management. Merge default project confi
  - :blue_heart: Typescript friendly
  - :muscle: Designed for multi-environment apps
  - :see_no_evil: Anonymized configuration for logger
+ - :eye_in_speech_bubble: JSONC support
```

Comment:
Alas, this does not seem to work 
![image](https://user-images.githubusercontent.com/24388536/70633942-a70f6280-1c31-11ea-9329-c598325715e4.png)

Reply:
ok GH supports eye_speech_bubble, GL supports eye_in_speech_bubble  🤯.

Can I use unicode for this one? :pray:

Reply:
Sure :+1:

Reply:
Fixed :checkered_flag: